plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMappingWithCacheDisabled.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.QueryRunner;
+
+public class TestBigQueryCaseInsensitiveMappingWithCacheDisabled
+        extends TestBigQueryCaseInsensitiveMapping
```

Comment:
Extending a non-abstract test class doesn't work well when we want to run a specific test on IDE. For instance, we can't run only `testNonLowerCaseSchemaName` with `TestBigQueryCaseInsensitiveMappingWithCacheDisabled` easily because Intellij doesn't show a popup menu to chose the test class.